it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/uaControllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -165,118 +168,207 @@ class CheckYourAnswersControllerISpec extends AuditEnabledSpecHelper
+
+      stubRetrieveEntityDetails(testJourneyId)(OK, testUAJourneyDataJson)
+      stubAudit()
+
+      val result = post(s"/identify-your-unincorporated-association/$testJourneyId/check-your-answers-business")()
+
+      result must have {
+        httpStatus(SEE_OTHER)
+        redirectUri(expectedValue = s"$testContinueUrl?journeyId=$testJourneyId")
       }
+
+      verifyStoreIdentifiersMatch(testJourneyId, expBody = JsString(SuccessfulMatchKey))
+      verifyCreateBusinessVerificationJourney(expectedBVJson)
+
+      verifyAudit()
     }
 
-    "redirect to the cannot confirm business error page" when {
-      "the identifiers are not matched" in {
-        enable(EnableFullUAJourney)
+    "identifiers are not matched, does not call BV and it redirects to the cannot confirm business error page. The journey is audited" in {
 
-        await(insertJourneyConfig(
-          journeyId = testJourneyId,
-          internalId = testInternalId,
-          testUnincorporatedAssociationJourneyConfig(businessVerificationCheck = true)
-        ))
+      enable(EnableFullUAJourney)
 
-        stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
-        stubRetrieveUtr(testJourneyId)(OK, testCtutrJson)
-        stubRetrievePostcode(testJourneyId)(OK, testOfficePostcode)
-        stubValidateUnincorporatedAssociationDetails(testCtutr, testOfficePostcode)(OK, Json.obj("matched" -> false))
-        stubStoreIdentifiersMatch(testJourneyId, DetailsMismatchKey)(OK)
-        stubRetrieveEntityDetails(testJourneyId)(OK, testUAJourneyDataJson)
-        stubAudit()
+      await(insertJourneyConfig(
+        journeyId = testJourneyId,
+        internalId = testInternalId,
+        testUnincorporatedAssociationJourneyConfig(businessVerificationCheck = true)
+      ))
 
-        val result = post(s"/identify-your-unincorporated-association/$testJourneyId/check-your-answers-business")()
+      stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+      stubRetrieveUtr(testJourneyId)(OK, testCtutrJson)
+      stubRetrievePostcode(testJourneyId)(OK, testOfficePostcode)
 
-        result must have {
-          httpStatus(SEE_OTHER)
-          redirectUri(expectedValue = errorControllers.routes.CannotConfirmBusinessController.show(testJourneyId).url)
-        }
+      stubValidateUnincorporatedAssociationDetails(testCtutr, testOfficePostcode)(OK, Json.obj("matched" -> false))
+      stubStoreIdentifiersMatch(testJourneyId, DetailsMismatchKey)(OK)
 
-        verifyStoreIdentifiersMatch(testJourneyId, expBody = JsString(DetailsMismatchKey))
-        verifyAudit()
+      stubStoreBusinessVerificationStatus(testJourneyId, expBody = testVerificationStatusJson(verificationStatusValue = "NOT_ENOUGH_INFORMATION_TO_CALL_BV"))(OK)
+      stubStoreRegistrationStatus(testJourneyId, RegistrationNotCalled)(OK)
+
+      stubRetrieveEntityDetails(testJourneyId)(OK, testNoIdentifiersJourneyDataJson)
```

Comment:
This doesn't look right. At the start the test we stub the retrieval of a utr and postcode, but later we are stubbing the retrieval of journey data with no identifiers.